src/common/types/store-data/scan-meta-data.ts

```diff
@@ -4,7 +4,7 @@ import { ToolData } from 'common/types/store-data/unified-data-interface';
 
 import { TargetAppData } from 'common/types/store-data/unified-data-interface';
 
-export type ScanMetaData = {
+export type ScanMetadata = {
```

Comment:
Should the file be updated to `scan-metadata.ts` for consistency with the new type name? (fine to leave this for a separate PR to split up review)

Reply:
Good point, I'll address it in a seperate pr